decidim-dev/decidim-dev.gemspec

```diff
@@ -28,7 +28,6 @@ Gem::Specification.new do |s|
   s.add_dependency "listen", "~> 3.1.0"
   s.add_dependency "launchy"
   s.add_dependency "i18n-tasks", "0.9.18"
-  s.add_dependency "faker", Decidim.faker_version
```

Comment:
Didn't we need this for seeds? 🤔

Reply:
Yes, but we're already specifying it in the `Gemfile` now. I figured that it's better to keep the dependency at a single place so it doesn't get out of sync easily.

In my opinion, though, `faker` should be a runtime dependency of `decidim`, since we are shipping stuff in `decidim-core`'s `lib/` that requires it.

Reply:
Either that, or be a little bit friendlier there by rescuing the `LoadError` of that require and point the user to install `faker` in the end application.

Reply:
Either way, completely unrelated to this PR, sorry about that. I wanted to introduce this change in #1958 but I forgot, so I put in here :flushed:.

Reply:
Nah, looks good to me. Still, we do have tests for seeds and app generation, so if this breaks something we'll catch it :)

Reply:
Note that this is a dependency of `decidim-dev`, which only gets (optionally) loaded if `decidim-dev` is on the gemfile. I think we need to keep it here and, in any case, maybe remove it from the `Gemfile`.

Is its removal related to bootsnap or just some cleanup?

Reply:
Yup, just some totally unrelated cleanup.

We had [problems in the past](https://github.com/decidim/decidim/issues/1709) because of having two separate version specifiers for `faker`, so I centralized the dependency under this `Decidim.faker_version` method. But we're no longer using it in the `Gemfile`, so the specifiers are out of sync again. So I figured we should remove one of those.

We can remove the one in the `Gemfile` but then people won't be able to seed easily in non-development environments (and we're telling them they can do it [here](https://github.com/decidim/decidim/blob/master/docs/getting_started.md#initializing-your-app-for-local-development)). That's why I think we should upgrade faker to a runtime dependency of `decidim` and all these issues would go away.

Reply:
@josepjaume I removed the change anyways, we can always bring it up separately if we agree on this!